src/dbnode/client/fetch_state_leakcheckpool_gen_test.go

```diff
@@ -26,11 +26,8 @@ package client
 
 import (
 	"fmt"
-
```

Comment:
These three `*_gen_test.go` files should have always had these extra blank lines removed during generation, but were not because the command `gen_cleanup_dir '*_gen.go'` have always omitted them.